### PR TITLE
Train corrector function

### PR DESCRIPTION
**Pull request by irinakhismatullina**
Small wrapper for generating trained TyposCorrector.

### CODE REVIEW

File: lookout/style/typos/preparation.py [hunk 1]
```diff
@@ -108,3 +109,31 @@ def prepare_data(params: Optional[Mapping[str, Any]] = None) -> pandas.DataFrame
     prepared_data = filter_splits(flat_data, vocabulary_tokens)[[Columns.Frequency, Columns.Split,
                                                                  Columns.Token]]
     return prepared_data
+
+
+def train_and_evaluate(train_data: pandas.DataFrame, test_data: pandas.DataFrame,
+                       vocabulary_path: str, frequencies_path: str, fasttext_path: str,
+                       threads_number: int = 8) -> TyposCorrector:
+    """
+    Create and train TyposCorrector model on given data.
```
Comment (vmarkovtsev) [inline lookout/style/typos/preparation.py]:
```suggestion
    Create and train TyposCorrector model on the given data.
```

File: lookout/style/typos/preparation.py [hunk 2]
```diff
@@ -108,3 +109,31 @@ def prepare_data(params: Optional[Mapping[str, Any]] = None) -> pandas.DataFrame
...
+
+    :param train_data: DataFrame, or its .csv dump, containing columns Columns.Token and
```
Comment (vmarkovtsev) [inline lookout/style/typos/preparation.py]: Should always be a dataframe only

File: lookout/style/typos/preparation.py [hunk 3]
```diff
@@ -108,3 +109,31 @@ def prepare_data(params: Optional[Mapping[str, Any]] = None) -> pandas.DataFrame
...
+                       Columns.CorrectToken, column Columns.Split is optional, but used
```
Comment (vmarkovtsev) [inline lookout/style/typos/preparation.py]:
```suggestion
                       Columns.CorrectToken, column Columns.Split is optional, but used \
```

File: lookout/style/typos/preparation.py [hunk 4]
```diff
@@ -108,3 +109,31 @@ def prepare_data(params: Optional[Mapping[str, Any]] = None) -> pandas.DataFrame
...
+                       when present.
+    :param test_data: Dataframe or its .csv dump, containing columns
```
Comment (vmarkovtsev) [inline lookout/style/typos/preparation.py]: same here: only dataframe

Comment (vmarkovtsev) [inline lookout/style/typos/preparation.py]:
```suggestion
    :param test_data: Dataframe or its .csv dump, containing columns \
```

**Comment by irinakhismatullina** [comment]
One build has failed for no reason, looks like it has nothing to do with the code, how to rerun it without pushing smth?

**Comment by vmarkovtsev** [comment]
Conflicts here and in #660 after merging #659

**Comment by vmarkovtsev** [comment]
> One build has failed for no reason, looks like it has nothing to do with the code, how to rerun it without pushing smth?

You should be able to restart the build in Travis. Usually it happens because of random networking problems.

**Comment by vmarkovtsev** [comment]
@irinakhismatullina Conflicts again :(

**Pull request merged**